### ISSUE TITLE
Re-import data from stable onto litterbox

### ISSUE THREAD

**Issue by abroddrick** [cisagov/manage.get.gov#2709]
### Issue description

During development, litterbox's database became reset, we should now try to re-import the data from stable on to litterbox.

### Acceptance criteria

- [ ] reimport data from stable onto litterbox 
- [ ] update any instructions that are now outdated

### Additional context

Everyone has the creds to do this, and this would be good for someone who is not dave to do. 

### Links to other issues

blocked by:
- #2862

**Comment by therealslimhsiehdy** [comment]
Note to self: Steps are listed in `import_export.md`

**Comment by therealslimhsiehdy** [comment]
Currently blocked due to import issues. 

* See [Slack discussion here](https://cisa-corp.slack.com/archives/C05BGB4L5NF/p1727290691121589)
* New ticket created that blocks this ticket [created here](https://github.com/cisagov/manage.get.gov/issues/2862)
* Draft PR for this ticket[ created here](https://github.com/cisagov/manage.get.gov/pull/2860)
